### PR TITLE
Project ui

### PR DESCRIPTION
**Pull request by ethanlow23**
I updated the styling for the upload receipt image(s) modal

upload-receipt-modal

### CODE REVIEW

File: client/src/pages/UploadReceipt.js [hunk 1]
```diff
@@ -1,37 +1,127 @@
...
+                    className={classes.drop}
+                    onDrop={(e) => {
+                      e.preventDefault();
+                      console.log(e);
+                    }}
+                    onDragOver={(e) => e.preventDefault()}
+                  >
+                    <CardMedia style={{marginLeft: 'auto', marginRight: 'auto'}} image={DragDropIcon} title="Drag and Drop Icon" className={classes.image} /><br />
+                    <Typography style={{margin: 12}} variant="h5" align="center">Drop files here</Typography>
+                    <Typography style={{margin: 12}} variant="body1" align="center">or</Typography>
```
Comment (bonnieli) [inline client/src/pages/UploadReceipt.js]: may be good to create a class for these 2 Typography cause they both have `margin: 12` and `align: center` if it's to center text, you can actually just assign this to the parent elem too!

File: server/api/receipt_controller.py [hunk 2]
```diff
@@ -58,7 +56,7 @@ def get_all_receipts(year, month):
         receipts = Receipt.query.all()
         # Receipt.query.all() returns empty list if there are no records
         if receipts == []:
-            return receipts
+            return jsonify({'receipts': receipts})
```
Comment (isabelt220) [inline server/api/receipt_controller.py]: I actually ended up removing this line in my branch cause the function returns total_amount = 0 and posts =[], so it was redundant to have this line. I'll be pushing my code onto the dev branch soon so you'll get the change later :)

Comment (ethanlow23) [reply server/api/receipt_controller.py]: thanks. sounds good!

**Review by isabelt220 — APPROVED** [review]
looks pretty good to me!

**Pull request merged**